src/plugins/dashboard/common/content_management/v2/cm_services.ts

```diff
@@ -34,6 +36,18 @@ export const dashboardAttributesSchema = dashboardAttributesSchemaV1.extends(
 
 export const dashboardSavedObjectSchema = savedObjectSchema(dashboardAttributesSchema);
 
+export const dashboardSearchOptionsSchema = dashboardSearchOptionsSchemaV1.extends(
+  {
+    sort: schema.maybe(
+      schema.object({
+        sortField: searchOptionsSchemas.sortField,
+        sortOrder: searchOptionsSchemas.sortOrder,
+      })
+    ),
+  },
+  { unknowns: 'ignore' }
+);
+
```

Comment:
I've added this search option in V2, as it's a backward compatible change and also opt-in I'm not sure if this warrants a V3